src/pudl/analysis/epacamd_eia.py

```diff
@@ -66,57 +66,42 @@ def _get_unique_keys(epacems: pd.DataFrame | dd.DataFrame) -> pd.DataFrame:
 def filter_crosswalk_by_epacems(
     crosswalk: pd.DataFrame, epacems: pd.DataFrame | dd.DataFrame
 ) -> pd.DataFrame:
-    """Inner join unique CEMS units with the EPA crosswalk.
+    """Inner join unique CEMS units with the epacamd_eia crosswalk.
 
     This is essentially an empirical filter on EPA units. Instead of filtering by
     construction/retirement dates in the crosswalk (thus assuming they are accurate),
     use the presence/absence of CEMS data to filter the units.
 
     Args:
-        crosswalk: the EPA crosswalk, as from pudl.output.epacems.epa_crosswalk()
+        crosswalk: epacamd_eia crosswalk
         unique_epacems_ids (pd.DataFrame): unique ids from _get_unique_keys
 
     Returns:
-        The inner join of the EPA crosswalk and unique epacems units. Adds
+        The inner join of the epacamd_eia crosswalk and unique epacems units. Adds
         the global ID column unit_id_epa.
 
     """
     unique_epacems_ids = _get_unique_keys(epacems)
     key_map = unique_epacems_ids.merge(
         crosswalk,
-        left_on=["plant_id_eia", "unitid"],
-        right_on=["CAMD_PLANT_ID", "CAMD_UNIT_ID"],
+        on=["plant_id_eia", "emissions_unit_id_epa"],
```

Comment:
So I'm assuming based on this that "emissions_unit_id_epa" equals the old "unitid" and not "unit_id_epa"? In CEMS has "unitid" been renamed "emissions_unit_id_epa" as well?

Reply:
Yes!